meteor-app/imports/ui/assessment/AssessmentWHOCarl.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import './AssessmentWHOCarlStyles.css'
+
+/*
+The app will have
+
+### Tabs:
+  1. 3 Tabbable pages
+    - Details
+    - Service Selector
+    - Summary (Hidden initially)
+        - only Render in the Admin panel to allow quick review
+          (Might need to check against the route as cannot use login)
+
+
+### Search bar:
+  1. A Search bar w drop down 
+    - items in drop down are clickable items.
+      - when clicked, the item will be added to an array
+      - The arr will be rendered to the screen with pills
+  2. Any active pills will be clickable to remove from array
+  3. When "misc" is added, The parts collection is now available to select from
+    - ** SHOULD BE A SEPERATE DROPDOWN.??  **
+
+
+### Price / Expected labour & items
+  1. This will be based off schema
+  2. render total price only.
+  3. Allow over-riding based on % off
+    - ** RETAIN BOTH FIGURES (BASE PRICE AND SOLD PRICE)??**
+
+### Major / Minor Service specific Pills
+  1. When either service is selected, all pills under that service are added automatically.
+    - when selected, it will grey out. Not delete.
+
+*/
+
+
+
+function todo({index, removeItem}){
+    return(
+        <div>
+            <button onClick={() => removeItem(index)}>X</button>
```

Comment:
You will need to be able to drag the button around, or the Full Service will expand on a click.